plans_test.go

```diff
@@ -33,12 +30,43 @@ func TestAccPlans(t *testing.T) {
 		// prints plans available in facility
 		log.Printf("%s: %+v\n", f, ps)
 	}
+}
 
-	if len(l) == 0 {
-		t.Fatal("Empty plans listing from the API")
+func TestAccPlansBasic(t *testing.T) {
+	skipUnlessAcceptanceTestsAllowed(t)
+
+	c, stopRecord := setup(t)
+	defer stopRecord()
+	l, _, err := c.Plans.List(&ListOptions{Includes: []string{"available_in"}})
+	if err != nil {
+		t.Fatal(err)
+	}
+	plansInFacilities(t, l)
+}
+
+func TestAccPlansProject(t *testing.T) {
+	skipUnlessAcceptanceTestsAllowed(t)
+	c, projectID, teardown := setupWithProject(t)
+	defer teardown()
+
+	l, _, err := c.Plans.ProjectList(projectID, &ListOptions{Includes: []string{"available_in"}})
+	if err != nil {
+		t.Fatal(err)
 	}
+	plansInFacilities(t, l)
+}
+
+func TestAccPlansOrganization(t *testing.T) {
+	skipUnlessAcceptanceTestsAllowed(t)
+
+	c, stopRecord := setup(t)
+	defer stopRecord()
+
+	user, _, err := c.Users.Current()
```

Comment:
linter:
```
plans_test.go:65:11: ineffectual assignment to `err` (ineffassign)
	user, _, err := c.Users.Current()
	```

Reply:
thanks, I fixed it. I also updated Makefile to make golangci-lint work out of the box and via docker in #235